.github/workflows/dependencies/dependencies_hip.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+#
+# Copyright 2020 The AMReX Community
+#
+# License: BSD-3-Clause-LBNL
+# Authors: Axel Huebl
+
+# search recursive inside a folder if a file contains tabs
+#
+# @result 0 if no files are found, else 1
+#
+
+set -eu -o pipefail
+
+# Ref.: https://rocmdocs.amd.com/en/latest/Installation_Guide/Installation-Guide.html#ubuntu
+wget -q -O - http://repo.radeon.com/rocm/rocm.gpg.key \
+  | sudo apt-key add -
+echo 'deb [arch=amd64] http://repo.radeon.com/rocm/apt/debian/ xenial main' \
```

Comment:
"xenial" is confusingly named: https://github.com/RadeonOpenCompute/ROCm/issues/1207
The "xenial" packages in ROCm 3.7 work only with bionic and focal.